client/hosting/server-settings/main.tsx

```diff
@@ -265,7 +302,7 @@ const ServerSettings = ( props ) => {
 				<HostingActivateStatus
 					context="hosting"
 					onTick={ requestUpdatedSiteData }
-					keepAlive={ ! isSiteAtomic && hasTransfer }
+					keepAlive={ Boolean( ! isSiteAtomic && hasTransfer ) }
```

Comment:
I think more right approach is to fix root cause instead of consequences with:

190 line - `!! transferState &&`

And
```suggestion
					keepAlive={ ! isSiteAtomic && hasTransfer }
```